### PR TITLE
[MINOR] Use SL4J string interpolation instead of string concatenation

### PR DESCRIPTION
**Pull request by dhruvilshah3**

### CODE REVIEW

**Comment by dhruvilshah3** [comment]
@hachikuji adding on to your pile of reviews ;-)
This one is only two lines of change so should hopefully be quick.

**Review by ijuma — APPROVED** [review]
LGTM. Have you checked that these are the only cases where we do this?

**Comment by dhruvilshah3** [comment]
@ijuma is the goal to look through the entire code base for such usage?

**Comment by ijuma** [comment]
No, I just meant in the PR where this was introduced.

**Comment by dhruvilshah3** [comment]
@ijuma I missed replying to this earlier. Those are the only two places with such usage in the chunked conversion PR.

File: clients/src/main/java/org/apache/kafka/common/record/LazyDownConversionRecordsSend.java [hunk 1]
```diff
@@ -63,7 +63,7 @@ public long writeTo(GatheringByteChannel channel, long previouslyWritten, int re
                             " converted_records_size: " + sizeOfFirstConvertedBatch);
 
                 recordConversionStats.add(recordsAndStats.recordConversionStats());
-                log.debug("Got lazy converted records for {" + topicPartition() + "} with length=" + convertedRecords.sizeInBytes());
+                log.debug("Got lazy converted records for topic-partition {} with length={}", topicPartition(), convertedRecords.sizeInBytes());
```
Comment (ijuma) [inline clients/src/main/java/org/apache/kafka/common/record/LazyDownConversionRecordsSend.java]: Nit: we typically just say `partition` in these cases. Same for the other `log.debug`.

File: clients/src/main/java/org/apache/kafka/common/record/LazyDownConversionRecordsSend.java [hunk 2]
```diff
@@ -75,8 +75,7 @@ public long writeTo(GatheringByteChannel channel, long previouslyWritten, int re
                 //      BaseOffset => Int64
                 //      Length => Int32
                 //      ...
-                // TODO: check if there is a better way to encapsulate this logic, perhaps in DefaultRecordBatch
-                log.debug("Constructing fake message batch for topic-partition {" + topicPartition() + "} for remaining length " + remaining);
+                log.debug("Constructing fake message batch for topic-partition {} for remaining length={}", topicPartition(), remaining);
                 int minLength = (Long.SIZE / Byte.SIZE) + (Integer.SIZE / Byte.SIZE);
```
Comment (ijuma) [inline clients/src/main/java/org/apache/kafka/common/record/LazyDownConversionRecordsSend.java]: I think what you want here is `Records.LOG_OVERHEAD`.

**Review by ijuma — COMMENTED** [review]
A couple of minor nits.

File: clients/src/main/java/org/apache/kafka/common/record/LazyDownConversionRecordsSend.java (hunk 2 above)
Comment (dhruvilshah3) [reply clients/src/main/java/org/apache/kafka/common/record/LazyDownConversionRecordsSend.java]: What we want to construct here is something that looks like a record batch. Do you think `Records.LOG_OVERHEAD` is appropriate for this case?

**Review by ijuma — APPROVED** [review]
LGTM

**Pull request merged**